CHANGELOG.md

```diff
@@ -72,7 +72,6 @@
 
 ### Fixes
 
-- Fixed styling of the incentivized testnet rewards wallet dropdown ([PR 2816](https://github.com/input-output-hk/daedalus/pull/2816/files))
```

Comment:
why remove an existing changelog entry instead of adding a new one for this PR?

Reply:
The change is being reverted and it was not released. It was basically a mistake that someone put it there in the changelog as it should be in vNext.

Reply:
@DominikGuzei this was a mistake, now we are correcting it